x-pack/plugins/observability_solution/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -353,19 +298,69 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
             spaceId,
             hostName: additionalContext?.host?.name,
           }),
-          originalAlertState: translateActionGroupToAlertState(originalActionGroup),
-          originalAlertStateWasALERT: originalActionGroup === FIRED_ACTIONS_ID,
-          originalAlertStateWasWARNING: originalActionGroup === WARNING_ACTIONS_ID,
           ...additionalContext,
+        };
+
+        const payload = {
+          [ALERT_REASON]: reason,
+          [ALERT_ACTION_GROUP]: actionGroup,
+          [ALERT_EVALUATION_VALUES]: evaluationValues,
+          [ALERT_EVALUATION_THRESHOLD]: thresholds,
```

Comment:
I tried a "between" comparator, which should be passing an array of values to thresholds.  The rule works fine, but this field is not set in the alert document.  For a single-value threshold (anything but between or not between), it printed the threshold fine.  

This may be a pre-existing issue with this rule type - I noticed that es query seems to not populate this field if it's multi-value as well, perhaps that's the pattern for these?

Reply:
I think this is intentional:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/infra/server/lib/alerting/common/get_values.ts#L34

Reply:
@pmuellr @ersin-erdal I've created a [ticket](https://github.com/elastic/kibana/issues/178067) for this topic to save the range threshold as an array. I also saw the the same behaviour in the es query rule, what do you think about adding the range threshold as an array in all rules (including es query)?

Reply:
That would be good, but I suspect we will have pre-existing data with just a scalar value - so any typing on such fields in Typescript will have to be `number | number[]`.